applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -133,6 +122,57 @@ static void le_audio_event_publish(enum le_audio_evt_type event, struct bt_conn
 	ERR_CHK(ret);
 }
 
+static void cap_start_worker(struct k_work *work)
+{
+	int ret;
+	uint8_t channel_index;
```

Comment:
Should this be stream index?

Reply:
No, but it could be device_index perhaps. Could also rename `headsets` --> `devices`

Reply:
Renamed `headsets` --> `unicast_servers` and changed `channel_index` --> `device_index`